apps/evm/src/hooks/useOperationModal/Modal/SupplyForm/index.tsx

```diff
@@ -113,18 +116,37 @@ export const SupplyFormUi: React.FC<SupplyFormUiProps> = ({
     [asset.vToken.underlyingToken, asset.vToken.address, chainId, isIntegratedSwapEnabled],
   );
 
+  const canWrapNative = useMemo(
```

Comment:
Nit - to be in line with other similar variables:
```suggestion
  const canWrapNativeToken = useMemo(
```